pkg/manage/cmd_config.go

```diff
@@ -0,0 +1,151 @@
+package manage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/OpenSlides/openslides-manage-service/pkg/datastore"
+	"github.com/OpenSlides/openslides-manage-service/proto"
+	"github.com/spf13/cobra"
+)
+
+const configHelp = `Get or set config values
+
+This command gets or sets the config values for an organisation.
+
+Example:
+
+$ manage config voting
+disabled
+$ manage config set voting enabled
+$ manage config voting
+enabled
+`
+
+// CmdConfig initializes the config command.
+func CmdConfig(cfg *ClientConfig) *cobra.Command {
+	values := []string{
+		"voting",
+	}
+
+	cmd := &cobra.Command{
+		Use:       "config",
+		Short:     "Get or set config values.",
+		Long:      configHelp,
+		Args:      cobra.ExactValidArgs(1),
+		ValidArgs: values,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx, cancel := context.WithTimeout(context.Background(), cfg.Timeout)
+			defer cancel()
+
+			service, close, err := Dial(ctx, cfg.Address)
+			if err != nil {
+				return fmt.Errorf("connecting to gRPC server: %w", err)
+			}
+			defer close()
+
+			req := &proto.ConfigRequest{
+				Field: proto.ConfigRequest_Field(proto.ConfigRequest_Field_value[args[0]]),
+			}
+
+			resp, err := service.Config(ctx, req)
+			if err != nil {
+				return fmt.Errorf("get config value %s: %w", args[0], err)
+			}
+
+			fmt.Println(resp.Value)
+			return nil
+		},
+	}
+
+	cmd.AddCommand(&cobra.Command{
+		Use:   "set",
+		Short: "Set a value",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if err := cobra.ExactArgs(2)(cmd, args); err != nil {
+				return err
```

Comment:
Here an extra fmt.Errorf() call would be nice.

Reply:
I copied that code from a cobra function.

I produces an error message for a user and not a developer. I would live it as it is.